idpcpp.cpp compilation issue in MacOSX

@PavithraNaullage reported failure to install with the following error:

```
src/idpconfgen/cpp/idpcpp.cpp:3981:37: error: non-constant-expression cannot be narrowed from type 'unsigned long' to 'long' in initializer list

[-Wc++11-narrowing]

std::vector<ssize_t> shape = {coords_with_sidechains.size() / 3, 3};

^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

src/idpconfgen/cpp/idpcpp.cpp:3981:37: note: insert an explicit cast to silence this issue

std::vector<ssize_t> shape = {coords_with_sidechains.size() / 3, 3};

^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

static_cast<long>( ) 
```

[Here's a possible solution](https://stackoverflow.com/questions/60368422/non-constant-expression-cannot-be-narrowed-from-type-int-to-unsigned-long-lon).

Proposed solution:

```cpp
ssize_t              ndim    = 2;
const int            scsize  = coords_with_sidechains.size(); 
std::vector<ssize_t> shape   = {scsize / 3, 3};
std::vector<ssize_t> strides = {sizeof(double) * 3 ,sizeof(double)}; 
```

I will commit this in a branch

--- a/src/idpconfgen/cpp/idpcpp.cpp
+++ b/src/idpconfgen/cpp/idpcpp.cpp
@@ -3978,7 +3978,8 @@ py::array faspr_sidechains( \
     std::vector<double> coords_with_sidechains = main_faspr(xyz, reslabel);
 
     ssize_t              ndim    = 2;
-    std::vector<ssize_t> shape   = {coords_with_sidechains.size() / 3, 3};
+    const int            scsize  = coords_with_sidechains.size();
+    std::vector<ssize_t> shape   = {scsize / 3, 3};
     std::vector<ssize_t> strides = {sizeof(double) * 3 ,sizeof(double)};
 
     // return 2-D NumPy array

